bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingReadLacOp.java

```diff
@@ -62,12 +66,12 @@
         this.cb = cb;
         this.numResponsesPending = lh.getLedgerMetadata().getEnsembleSize();
         this.coverageSet = lh.distributionSchedule.getCoverageSet();
+        this.currentEnsemble = lh.getCurrentEnsemble();
     }
 
     public void initiate() {
-        LedgerMetadata metadata = lh.getLedgerMetadata();
-        for (int i = 0; i < metadata.currentEnsemble.size(); i++) {
-            lh.bk.getBookieClient().readLac(metadata.currentEnsemble.get(i),
+        for (int i = 0; i < currentEnsemble.size(); i++) {
+            lh.bk.getBookieClient().readLac(currentEnsemble.get(i),
```

Comment:
This is perfect to me

Reply:
@eolivelli can you elaborate why you made 'perfect' comment just for this? Just curious. :)